L/libjulia/libjulia@1.6/bundled/patches/0001-flisp-system.patch

```diff
@@ -0,0 +1,40 @@
+From 30c32040d81ba909242b6e5786772efa94e8120f Mon Sep 17 00:00:00 2001
+From: Valentin Churavy <v.churavy@gmail.com>
+Date: Wed, 23 Sep 2020 16:48:49 -0400
+Subject: [PATCH] Allow flips to be built against system UTF8PROC
```

Comment:
Is this patch already in Julia master, or at least in a PR there? Perhaps it could alao be backported to the release-1.6 branch?

Reply:
I'm confused as well: the patch applies cleanly, but the [corresponding PR](https://github.com/JuliaLang/julia/pull/37723) was merged and appears to [have been backported](https://github.com/JuliaLang/julia/pull/38949) to v1.6

Reply:
Uh, it looks like [this commit](https://github.com/JuliaLang/julia/pull/37723/commits/ba653ecb1c81f1465505c2cea38b4f8149dd20b3) was skipped from the backport